src/dpp/cluster/webhook.cpp

```diff
@@ -73,12 +73,14 @@ void cluster::execute_webhook(const class webhook &wh, const struct message& m,
 		{"with_components", true},
 	});
 	std::string body;
-	if (!thread_name.empty() || !wh.avatar.to_string().empty() || !wh.name.empty()) { // only use json::parse if thread_name is set
+	if (!thread_name.empty() || !wh.avatar.to_string().empty() || !wh.avatar_url.empty() || !wh.name.empty()) { // only use json::parse if thread_name is set
```

Comment:
it might be nice to tidy up the comment here while we're in this bit of code. It was not correct before, and certaintly isnt now :-)